src/ethaniccc/Mockingbird/processing/InboundPacketProcessor.php

```diff
@@ -293,11 +293,11 @@ public function process(DataPacket $packet, User $user) : void{
                         $this->handleClick($user);
                         break;
                     case InventoryTransactionPacket::TYPE_USE_ITEM:
-                        switch($packet->trData->actionType){
+                        switch($packet->actionType){
```

Comment:
Is `$packet->trData->getActionType()`

Reply:
'$packet->trData->getActionType()' is error...
IThis is what I tried to fix. But that code also throws an error.